### PR TITLE
adding exe files and .vscode to .gitignore

### PR DESCRIPTION
**Pull request by p0l0us**
### Purpose

Ignore *.exe files by git.

After PR discussion .idea and .vscode removed from .gitignore.

### Testing

Tested using git.

### Documentation

No changes.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,7 @@ stdiscosrv.exe
 *.zip
 *.asc
 *.deb
+*.exe
```
Comment (calmh) [inline .gitignore:9]: Maybe remove the other two exe patterns above

File: .gitignore [hunk 2]
```diff
@@ -20,3 +21,4 @@ deb
 /proto/scripts/protoc-gen-gosyncthing
 /gui/next-gen-gui
 .idea
+.vscode
```
Comment (calmh) [inline .gitignore:24]: Let's not add this; the .idea one is a historical mistake, and we shouldn't have to track which editors exist

Comment (p0l0us) [reply .gitignore]: On other hand it is making life or developers easy. I don't think there are that many editors widely used. But if you wish...

Comment (acolomb) [reply .gitignore]: There is still the possibility to add local ignore patterns in Git's config. Suitable for every programme's own taste and tooling. Thus I agree we should keep all those dort-directories out of the committed ignore list.

**Review by calmh — APPROVED** [review]

**Pull request merged**